tests/integration_tests/tests/test_cases.py

```diff
@@ -788,11 +788,14 @@ def tearDown(self):
         self.examples.cleanup()
         super(PluginsTest, self).tearDown()
 
-    @staticmethod
-    def get_wagon_path(plugin_path):
+    # @staticmethod
+    def get_wagon_path(self, plugin_path):
```

Comment:
it returns a list now, why that can happen?

Reply:
We have wagons for more than one platform (`Centos`, `RedHat`).